cost-analyzer/templates/kubecost-metrics-deployment-template.yaml

```diff
@@ -0,0 +1,266 @@
+{{- if .Values.kubecostMetrics }}
+{{- if .Values.kubecostMetrics.exporter }}
+{{- if or .Values.kubecostMetrics.exporter.enabled .Values.agent }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "kubecost.kubeMetricsName" . }}
+  labels:
+    app: {{ template "kubecost.kubeMetricsName" . }}
+{{- with .Values.kubecostMetrics.exporter.labels }}
+{{ toYaml . | indent 4 }}
+{{- end }}
+spec:
+  replicas: {{ .Values.kubecostMetrics.exporter.replicas | default 1 }}
+  selector:
+    matchLabels:
+      app: {{ include "kubecost.kubeMetricsName" . }}
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: {{ template "kubecost.kubeMetricsName" . }}
+        {{- if .Values.global.additionalLabels }}
+        {{ toYaml .Values.global.additionalLabels | nindent 8 }}
+        {{- end }}
+{{- with .Values.kubecostMetrics.exporter.labels }}
+{{ toYaml . | indent 8 }}
+{{- end }}
+{{- if .Values.global.podAnnotations }}
+      annotations:
+{{- with .Values.global.podAnnotations }}
+{{ toYaml . | indent 8 }}
+{{- end }}
+{{- end }}
+    spec:
+      restartPolicy: Always
+      serviceAccountName: {{ template "cost-analyzer.serviceAccountName" . }}
+      volumes:
+        {{- if .Values.agent }}
+        - name: config-store
```

Comment:
`config-store` seems to be a generic name which doesn't mean much to me, what is it supposed to contain?

Reply:
Mounting the store configuration for shared resources in S3/GCR.